_posts/2021-04-14-release-statefun-3.0.0.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title:  "Stateful Functions 3.0.0 Release Announcement"
+subtitle: "The Apache Flink community is happy to announce the release of Stateful Functions (StateFun) 3.0.0."
+date: 2021-04-14T08:00:00.000Z
+categories: news
+authors:
+- igalshilman:
+  name: "Igal Shilman"
+  twitter: "IgalShilman"
+- tzulitai:
+  name: "Tzu-Li (Gordon) Tai"
+  twitter: "tzulitai"
+---
+
+The Apache Flink community is happy to announce the release of Stateful Functions (StateFun) 3.0.0!  This release focuses on bringing remote functions to the front and center of StateFun. It is now easier, more efficient, and more ergonomic to write applications that live in a separate process to the StateFun cluster.
+
+The binary distribution and source artifacts are now available on the updated [Downloads](https://flink.apache.org/downloads.html)
```

Comment:
Can we move these more "technical" announcement further down in the blog post (link to artifacts and release notes)?

Reply:
+1

Reply:
Moved this paragraph to a `Release Resources` section at the very end.